code/app/src/main/java/com/adobe/marketing/tester/TestApplication.java

```diff
@@ -12,20 +12,21 @@
 package com.adobe.marketing.tester;
 
 import android.app.Application;
-import android.util.Log;
 import com.adobe.marketing.mobile.Assurance;
 import com.adobe.marketing.mobile.Edge;
 import com.adobe.marketing.mobile.LoggingMode;
 import com.adobe.marketing.mobile.MobileCore;
 import com.adobe.marketing.mobile.edge.consent.Consent;
 import com.adobe.marketing.mobile.edge.identity.Identity;
+import com.adobe.marketing.mobile.services.Log;
 import java.util.Arrays;
 
 public class TestApplication extends Application {
 
-	private static final String LOG_TAG = "TestApplication";
+	private static final String LOG_TAG = "EdgeTestApplication";
```

Comment:
MainActivity is using `LOG_TAG = "TestApp"`. Can you consolidate the two? Maybe rename to "EdgeTestApp", make `public static final` and then static import this LOG_TAG into MainActivity?